src/components/Comparisons/RelatedItem.jsx

```diff
@@ -44,14 +44,24 @@ function RelatedItem (props) {
   const [image, setImage] = useState();
 
 
-  // gets styles of the related product and adds the thumbnail picture to the image source
-  // needs to loop through styles and get the default picture
+  // gets styles of the related product and adds the default thumbnail picture to the image source
+  // if there is no default style, it grabs the first picture in the list
   useEffect(() => {
+    var thumbnail = '';
     if (props.item) {
       server.get('/products/styles', {product_id: props.item.id})
         .then((data)=> {
-          var thumbnail = data.data.results[0].photos[0].thumbnail_url
-          setImage(thumbnail);
+          var results = data.data.results;
+          for (var i = 0; i < results.length; i++) {
+            if (results[i]['default?'] === true) {
+              thumbnail = results[i].photos[0].thumbnail_url;
+              setImage(thumbnail);
+            }
+          }
+          if (thumbnail === '') {
```

Comment:
Curious why this is needed? Do some items not have a listed default I'm guessing? (asking just for curiousity's sake)

Reply:
Yes. Product 66648 has no styles set as default?: true. they're all in stock but all say default: false. Not sure why!